rollup.config.js

```diff
@@ -19,6 +19,28 @@ const rollup = require('rollup');
 const resolve = require('rollup-plugin-node-resolve');
 const cleanup = require('rollup-plugin-cleanup');
 
+// Some third party libraries may have optional modules.
+// Ignore the warnings generated by rollup for expected
+// missing modules.
+const IGNORE_MISSING = {
+  'lib/third_party/three/GLTFLoader.js': [
+    'DDSLoader',
+    'BufferGeometryUtils',
+    'DRACOLoader',
```

Comment:
Will we eventually include this for draco support?

Reply:
Yeah, if we want to bundle it, but imagine we'd need a different solution. This expects `THREE.DRACOLoader` so to avoid changing the source code that will get clobbered on the next three.js update, the warnings are muted. For DRACO specifically, I think we can set that in CachedGLTFLoader, rather than modifying GLTFLoader outside of the ES Module changes